tools/pymanifold.cpp

```diff
@@ -0,0 +1,169 @@
+#include "manifold.h"
+#include "meshIO.h"
+#include "pybind11/functional.h"
+#include "pybind11/numpy.h"
+#include "pybind11/operators.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+namespace py = pybind11;
+
+using namespace manifold;
+
+void exportManifold(Manifold &m, std::string name) {
+  Mesh out = m.GetMesh();
+  manifold::ExportOptions options;
+  options.faceted = true;
+  options.mat.roughness = 0.2;
+  options.mat.metalness = 0.0;
+  manifold::ExportMesh(name, out, options);
+};
+
+typedef std::tuple<float, float> Float2;
+typedef std::tuple<float, float, float> Float3;
+
+struct PolygonsWrapper {
+  std::unique_ptr<Polygons> polygons;
+};
+
+PYBIND11_MODULE(pymanifold, m) {
+  m.doc() = "Python binding for the manifold library. Please check the C++ documentation for APIs.\n"
+      "This binding will perform copying to make the API more familiar to OpenSCAD users.";
+  py::class_<Manifold>(m, "Manifold")
+      .def(py::init<>())
+      .def(py::init([](std::vector<Manifold> &manifolds) {
+        Manifold result;
+        for (Manifold &manifold : manifolds)
+          result += manifold;
+        return result;
+      }), "Construct manifold as the union of a set of manifolds.")
+      .def(py::self + py::self)
+      .def(py::self - py::self)
+      .def("__and__", [](Manifold &a, Manifold &b) { return a ^ b; })
```

Comment:
I take it python doesn't let us overload the `^` operator?

Reply:
It allows us to overload that, but I thought intersection is more similar to and, and was using this in my previous code and forgot to change. Will change that back to `^`.